airflow/cli/commands/info_command.py

```diff
@@ -279,6 +285,21 @@ def __str__(self):
         )
 
 
+class ProvidersInfo:
+    """providers information"""
+
+    def __str__(self):
+
+        tabulate_data = [
+            {
+                'Provider name': provider['package-name'],
+                'Version': provider['versions'][0],
+            }
+            for version, provider in ProvidersManager().providers.values()
```

Comment:
@potiuk , better to have this `version` replaced by `_` given it is not used here? Or if I missed anything? of course it's a nit.

Reply:
Or `provider['versions'][0]` above should be replaced by `version` from this line?
Based on my understanding from https://github.com/apache/airflow/blob/master/airflow/providers_manager.py#L169-L171

May be wrong though of course, and as mentioned above, this is minor

Reply:
@XD-DENG I'm working on this

Reply:
Great 👍

Reply:
@XD-DENG - > True, good eye, We need to still implement final versioning information for providers. And it is simple (but not minor :). I think it is critical before we release 2.0.

For now this information might be in three places: we have list of versions in provider.yaml (for documentation purpose) and we have version of currently installed version in the provider package, and (for now) all the versions for all provider packages are hard-coded when released (to 1.0.0b2). This has to be synced-up automatically and we have to describe how to update the versions when you add changes to provider (and whethere to increase majonr/minor/patchlevel for it).  

And it should work in both cases - when we run Airlfow directly from sources during development and unit tests (there provider.yaml files are read) or when we are running in production and provider is installed from the packages (then we read version information from the package). For now I just take versions[0] as this information is available in all cases (but it might be not accurate and for example might not match the version of the package).

The issue for that is here: https://github.com/apache/airflow/issues/11425 - which we still need to complete it before 2.0.0rc1 - it is mostly about making sure that package versions in provider.yaml and the version that is currently being released are in sync and that we know how to update it in-the future, separately for each package.

Not complex but important :).